components/authz-service/storage/postgres/migration/sql/54_use_project_db_id_in_references_of_project_rules.up.sql

```diff
@@ -0,0 +1,313 @@
+BEGIN;
+-- break all references
+ALTER TABLE iam_project_rules
+    DROP CONSTRAINT iam_project_rules_project_id_fkey;
+ALTER TABLE iam_staged_project_rules
+    DROP CONSTRAINT iam_staged_project_rules_project_id_fkey;
+
+-- When we're done fixing all references, we'll change the
+-- primary key of iam_projects; now, we only add a uniqueness
+-- constraint on what's going to be the new PK: db_id.
+ALTER TABLE iam_projects
+    ADD UNIQUE (db_id);
```

Comment:
This allows us to update the references piece-by-piece. When we're done with the other tables, we can finally
```
ALTER TABLE iam_projects
    DROP CONSTRAINT iam_projects_pkey;
ALTER TABLE iam_projects
    ADD PRIMARY KEY (db_id);
ALTER TABLE iam_projects
    ADD UNIQUE (id);
```